### ISSUE TITLE
I wonder to know what the loss number of your training  when you stop training will be? 

### ISSUE THREAD

**Issue by MingPap** [AlexeyAB/darknet#2085]
@AlexeyAB  Hi~ 
I have trained my dataset for about 17000 iterations. I have 3 class. the training picture size is 1224*370

but the loss is still around 0.7 and decrease very slowly

the learning rate in my cfg file is 0.001 as default, should I increase the learning rate and keep training?

here is my cfg file

[yolov3-fdepth.txt](https://github.com/AlexeyAB/darknet/files/2701464/yolov3-fdepth.txt)

with your kindness. help wanted!
